test.js

```diff
@@ -2,6 +2,33 @@ var assert = require( "assert" );
 var zlib = require( "zlib" );
 var s = require( "./index" );
 var type;
+let expectedCSVData = [
```

Comment:
For compatibility (and also for being consistent with the rest of the code) I think we should avoid ES6 features like `let` here, so this library can continue to be used with older versions of Node